api/graphql/graphql_test.go

```diff
@@ -81,19 +81,6 @@ func TestQueries(t *testing.T) {
                 }
               }
       
-              comments(first: 2) {
-                pageInfo {
-                  endCursor
-                  hasNextPage
-                  startCursor
-                  hasPreviousPage
-                }
-                nodes {
-                  files
-                  message
-                }
-              }
-      
```

Comment:
This must be fixed. See following commit messages:

- 4fdc3c77aac1c2d0f36e13dc27b270faad445601
- 378120409ac8877f98916f7b8bcdb18302d7b83d

Reply:
Fixed by commit b0e7bc42e60929c5f27b3c78ab07efce7e480915